### DISCUSSION TITLE
Android Security for PKI Admin PAYLOADVARIANT_NOT_SET

### DISCUSSION THREAD

**Discussion by dwright1542** [meshtastic/Discussions#9] [Android]
When using the Android app to try to turn on PKI Admin, if I click on the Security tab, I get a PAYLOADVARIANT_NOT_SET error.    Firmware is 2.5

**Comment by Droopydroors** [comment]
I think i saw that before if i was using firmware on the node that was prior to 2.5.  So basically the app has the option but your node does not because its not running 2.5 firmware.

**Reply by dwright1542** [reply]
running 2.5
